metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/InfluxDbReporter.java

```diff
@@ -136,10 +138,22 @@ public Builder includeMeterFields(Set<String> fields) {
             return this;
         }
 
+        /**
+         * Map measurement to a defined measurement name, where the key is the measurement name
+         * and the value is the reqex the measurement should be mapped by.
+         *
+         * @param measurementMappings
+         * @return {@code this}
+         */
+        public Builder measurementMappings(Map<String, String> measurementMappings) {
```

Comment:
To make it easy to use, probably construct the Map<String, Pattern> here and throw a runtime if a pattern doesn't compile?